Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION = 1.0.0
+VERSION = 1.0.1
```

Comment:
I'd be inclined to say this should be a minor release - mainly due to the addition of the new field in the CRD. A patch/bug release ideally wouldn't introduce a new field.

Reply:
I'll push a new branch to name it correctly and close out this PR.